v2/googlecloud-to-elasticsearch/src/test/java/com/google/cloud/teleport/v2/elasticsearch/templates/GCSToElasticsearchTest.java

```diff
@@ -95,8 +95,8 @@ public void testGCSToElasticsearchUdfE2E() {
                 "ConvertLine",
                 CsvConverters.LineToFailsafeJson.newBuilder()
                     .setDelimiter(options.getDelimiter())
-                    .setUdfFileSystemPath(options.getJavascriptTextTransformGcsPath())
-                    .setUdfFunctionName(options.getJavascriptTextTransformFunctionName())
+                    .setJavascriptUdfFileSystemPath(options.getJavascriptTextTransformGcsPath())
+                    .setJavascriptUdfFunctionName(options.getJavascriptTextTransformFunctionName())
```

Comment:
Same here - adding some python udf variants for coverage.

Reply:
added